pkg/inst/worker/worker.R

```diff
@@ -1,5 +1,24 @@
 # Worker class
 
+# Utility function to get current system time in milliseconds
+# this code snippet borrowed from [R.utils](cran.r-project.org/package=R.utils)
+currentTimeMillis <- function() {
```

Comment:
Well all the cases in this file we only need relative time (i.e. time difference) -- so there shouldn't be much of a difference if we use milliseconds since epoch vs. milliseconds in this process.